### DISCUSSION TITLE
Multiple UDP Packets

### DISCUSSION THREAD

**Discussion by xilinx8** [dannagle/PacketSender#226] [Q&A]
#### What OS?

- [X] Windows
- [ ] Mac
- [ ] Linux (Which distro?)

#### Description of issue
I just started using Packetsender. I need to send two consecutive packets to the same desitination ip. Is there a way to do that with a single send command with a specified delay in between. How can I use the file option. Is it possible to load the file option with multiple packets? I looked up the documentation, but could not find any detailed documentation on your website. Thanks

**Comment by dannagle** [comment]
Hello, the current major version of Packet Sender (version 7) does not have a single-click -> send-multiple packets feature, but the next major release (version 8) does support this kind of operation. 

I don't have any official binaries for version 8 posted because it is still in development. Would you be willing to supply me with early feedback if I make an alpha build for you?  If yes, please email hello@packetsender.com 

For your second question... "Load File" simply reads a file and HEX decodes it for sending. There is a packet database export/import feature if you want to transfer packets (or you can save packets to Packet Sender Cloud).

**Comment by xilinx8** [comment]
Hi Dan,
Thank you for your email. I would be happy to provide you feedback provided I do not need to send any specifics on my project.
Thanks,
Yaser

From: Dan Nagle ***@***.***>
Sent: Thursday, March 18, 2021 9:18 PM
To: dannagle/PacketSender ***@***.***>
Cc: Yaser Khalifa ***@***.***>; Author ***@***.***>
Subject: [External] Re: [dannagle/PacketSender] Multiple UDP Packets (#226)


Hello, the current major version of Packet Sender (version 7) does not have a single-click -> send-multiple packets feature, but the next major release (version 8) does support this kind of operation.

I don't have any official binaries for version 8 posted because it is still in development. Would you be willing to supply me with early feedback if I make an alpha build for you? If yes, please email ***@***.******@***.***>

For your second question... "Load File" simply reads a file and HEX decodes it for sending. There is a packet database export/import feature if you want to transfer packets (or you can save packets to Packet Sender Cloud).

**Reply by dannagle** [reply]
Hello, I never received your email. Please email address at https://packetsender.com/contact if you'd like a copy and help test. 

I plan to work on this feature after the next release (coming soon).

**Comment by denizci88** [comment]
Hello,

Thank you for Packet Sender. We are looking forward to seeing this useful feature in Packet Sender.

**Comment by SimonMerrett** [comment]
Yes, I too would be very grateful to see this feature included in Packet Sender. Thanks for all your work Dan!
